freediscovery/tests/test_stop_words.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import os
+import os.path
+import shutil
+
+from unittest import SkipTest
+from freediscovery.stop_words import (_StopWordsWrapper)
+
+
+basename = os.path.dirname(__file__)
+current_point = os.getcwd()
+
+if os.name == 'nt': # on windows
+    cache_dir = os.getcwd() + '\\acsw\\'
+else:
+    cache_dir = os.getcwd() + '/acsw/'
+
+custom_sw = _StopWordsWrapper(cache_dir = cache_dir)
+
+stop_words_for_test = ['one', 'two', 'three']
```

Comment:
Let's call this `tested_stop_words` (to avoid prepositions 'in', 'for', 'a', 'the' etc in variable names)..